vscode-extension/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
```

Comment:
Is this by convention for vscode extension? Generally I use es6 instead of commonjs. But it doesn't matter much really.

Reply:
This was preset as I followed the steps on the extension guide. So I'd rather leave it as is.